ts/components/LabelledItem/index.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * Display a labelled, followed by a
+ * input and an icon on the left-end
+ * side of the input
+ *
+ * LABEL
+ * X __________
+ * ^     ^
+ * icon  |
+ *       input
+ */
+import { Item, View } from "native-base";
+import * as React from "react";
+import { useState } from "react";
+import {
+  ImageSourcePropType,
+  ImageStyle,
+  StyleSheet,
+  TextInputProps
+} from "react-native";
+import { TextInputMaskProps } from "react-native-masked-text";
+import { IconProps } from "react-native-vector-icons/Icon";
+import { NavigationEvents } from "react-navigation";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+import { WithTestID } from "../../types/WithTestID";
+import { H5 } from "../core/typography/H5";
+import TextInputMask from "../ui/MaskedInput";
+import { Icon } from "./Icon";
+import { Input } from "./Input";
+
+const styles = StyleSheet.create({
+  noBottomLine: {
+    borderBottomWidth: 0
+  },
+  bottomLine: {
+    borderBottomWidth: 1
+  }
+});
+
+// Style type must be generalized so as to support both text icons and image icons
+type StyleType = IconProps["style"] & ImageStyle;
+
+type CommonProp = Readonly<{
+  label?: string;
+  icon?: string | ImageSourcePropType;
+  iconPosition?: "left" | "right";
+  iconStyle?: StyleType;
+  iconColor?: string;
+  isValid?: boolean;
+  focusBorderColor?: string;
+  description?: string;
+  accessibilityLabel?: string;
+  accessibilityHint?: string;
+  accessibilityLabelIcon?: string;
+  onPress?: () => void;
+  hasNavigationEvents?: boolean;
+}>;
+
+interface InputProps extends TextInputProps {
+  disabled?: boolean;
+}
+
+type Props = WithTestID<CommonProp> &
```

Comment:
Same as above. It makes sense to reuse what is [already defined](https://github.com/pagopa/io-app/pull/3143/files#diff-8c07de8bb4b02bcb2719f5034c992632402e81914e1c1ae479d655398a176613R25) by the type's owner.